### ISSUE TITLE
aliasing problem in reprojected image

### ISSUE THREAD

**Issue by LevinJ** [DRosemei/RoMe#41]
In the blend image generated during Rome evaluation process, there are very obvious aliasing problems(锯齿), especially on the reprojeted blue lane lines. On the other side, the reconstructed Semantic BEV has no obvious aliasing, as shown in below images. Does anyone know how this aliasing are formed and how they can be eliminated? Thanks a lot in advances.

![bev_seg_7_6e71a71238ab0a8852b4](https://github.com/DRosemei/RoMe/assets/1629014/13f865e1-5dbe-430c-9cf5-366635849c41)
![blend_image_7_bc2f83446b6f8a59e25f](https://github.com/DRosemei/RoMe/assets/1629014/4bd849ae-de15-4c8d-aa11-72fe94c92d27)

**Comment by DRosemei** [comment]
It is due to bev resolution. You can set lower bev_resolution, like 0.05 or 0.02 for better alignment, but it will cost larger resources.

**Comment by LevinJ** [comment]
Yes, it's indeed because of bev resolution. Checking the generated mesh carefully, I can  see the jaggies in the shape of mesh triangles. After setting it to 0.05, I am able to get much improved result. Thanks!
![eval_00064-blend](https://github.com/DRosemei/RoMe/assets/1629014/069f4dac-ea6c-499d-b7e1-9dba5900e90c)
